apps/core/dynamodb-local-dir.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const os = require('os');
+const path = require('path');
+
+// reference to https://github.com/rynop/dynamodb-local/blob/1cca305c077bd600ad972569e42647d17782e921/index.js#L17
+console.log(path.join(os.tmpdir(), 'dynamodb-local'));
```

Comment:
Clever use of console.log + rimraf! Cool! You may want to add a comment explaining how this log is being used to make it more obvious to readers. I honestly didn't know you could do something like this. It's like you're piping data to stdout and reading it back in.

Reply:
added a block to explain what this script does 🙏

Reply:
Thank you!